service/src/main/scala/com/yahoo/maha/service/utils/MahaRequestLogUtils.scala

```diff
@@ -164,7 +164,7 @@ case class MahaRequestLogHelper(mahaRequestContext: MahaRequestContext, mahaRequ
     val drivingQuery = queryPipeline.queryChain.drivingQuery
     val model = queryPipeline.queryChain.drivingQuery.queryContext.requestModel
     val factBestCandidateOption = queryPipeline.factBestCandidate
-    val engine = MahaRequestProto.Engine.Hive
+    val engine = queryPipeline.queryChain.drivingQuery.engine
```

Comment:
Need to convert type of the engine from maha core to MahaRequestProto Engine type